image/mkosi.conf.d/selinux.conf

```diff
@@ -0,0 +1,3 @@
+[Output]
+# set selinux to permissive
+KernelCommandLine=!selinux=0 selinux=1 enforcing=0
```

Comment:
What does `!selinux=0` do?

Reply:
mkosi prefills the kernel commandline with `selinix=0`. This removes that default before appending `selinux=1`.

> To remove specific arguments add a space separated list of "!" prefixed arguments.
https://github.com/systemd/mkosi/blob/main/mkosi.md

Reply:
Oh, it's from mkosi. Alright.